src/jitter.rs

```diff
@@ -21,47 +25,106 @@ mod sys {
         JITTER_BUFFER_GET_LATE_COST = 13,
     }
 
-    #[derive(Clone, Copy, Debug)]
+    #[derive(Clone, Copy, Debug, PartialEq)]
     pub enum Error {
-        FailedInit,
-        UnknownRequest,
+        BufferOk = 0,
+        BufferMissing = 1,
+        BufferInsertion = 2,
+        BufferInternalError = -1,
+        BufferBadArgument = -2,
     }
 
     impl fmt::Display for Error {
         fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
             let v = match self {
-                Error::FailedInit => "Failed to initialize",
-                Error::UnknownRequest => "The request is unknown",
+                Error::BufferOk => "The buffer is ok",
+                Error::BufferMissing => "The buffer is missing",
+                Error::BufferInsertion => "Bad insertion into the buffer",
+                Error::BufferInternalError => "Internal Error",
+                Error::BufferBadArgument => "Bad Argument for the buffer",
```

Comment:
nit: to be consistent with case, I'd do "Internal error" and "Bad argument..." here

Reply:
Done. Thanks! :)